src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -251,8 +251,8 @@ internal sealed class SourcePropertySymbol : PropertySymbol, IAttributeTargetSym
 
                         // Although we only do this in error scenarios, it is undesirable to mutate the symbol by setting its type twice.
```

Comment:
>// Although we only do this in error scenarios, it is undesirable to mutate the symbol by setting its type twice. [](start = 24, length = 113)

This comment looks incorrect, this is not an error scenario, this is a success case.